scripts/extensions/sams/src/components/sets/setListItem.tsx

```diff
@@ -58,7 +59,7 @@ export class SetListItem extends React.PureComponent<IProps> {
                 <ListItemBorder />
                 {set.state === SET_STATE.DRAFT ? null : (
                     <ListItemColumn>
-                        <Badge text="0" type="success" />
+                        <Badge  text = {count.toString()} type="success" />
```

Comment:
I'm not sure why our lint rules didn't pick this up, could you remove the spaces around the `=` in this assignment along with the extra space before it (just to keep a standard with the rest of the codebase)